capture/main.c

```diff
@@ -43,6 +43,7 @@ unsigned char          moloch_hex_to_char[256][256];
 
 extern MolochWriterQueueLength moloch_writer_queue_length;
 extern MolochPcapFileHdr_t     pcapFileHeader;
+uint32_t                hashSalt;
```

Comment:
can you move all of this to session.c and make it a LOCAL var just for session.c